src/applications/financial-status-report/components/alerts/ReviewPageNavigationAlert.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+import { setData } from 'platform/forms-system/src/js/actions';
+
+const ReviewPageNavigationAlert = ({ data, title }) => {
+  const dispatch = useDispatch();
+  const onReviewClick = () => {
+    dispatch(
+      setData({
+        ...data,
+        reviewNavigation: false,
+      }),
+    );
+  };
+
+  return (
+    <va-alert
+      class="vads-u-margin-y--1"
+      status="info"
+      data-testid="review-page-navigation-alert"
+      uswds
```

Comment:
Hey @Dr-Pongo, can you make a couple small UI tweaks to the alert? 

Can you add the `background-only` prop and change class to `vads-u-margin-bottom--2`?